apps/vast/vast-bridge.py

```diff
@@ -210,16 +286,15 @@ async def match_intel(
     """
     Reads from the intel_queue and matches all IoCs, either via VAST's
     live-matching or retro-matching.
-    @param cmd The vast binary command to use with PyVAST
+    @param vast_cmd The vast binary command to use with PyVAST
```

Comment:
```suggestion
    @param vast_binary The vast binary command to use with PyVAST
```

I find `vast_binary` clearer. With `_cmd`, I associate a VAST-specific command, like `export` or `import`.